### PR TITLE
inets, stdlib: move rfc1123_date and related functions from httpd_uti…

### PR DESCRIPTION
**Pull request by ao-song**
…l to calendar

Change-Id: Id45e4004ad1e39cdd93ba181607dcaf3e22b2528

### CODE REVIEW

**Comment by garazdawi** [comment]
Shouldn't the usage of the httpd_util functions first be deprecated before removed?

**Comment by ao-song** [comment]
> Shouldn't the usage of the httpd_util functions first be deprecated before removed?

I think you are right, the functions should be deprecated first before totally moved to another module.

**Comment by IngelaAndin** [comment]
You can move them and deprecate them at the same time. That is you must keep wrapper functions in httpd_util for now, but they can be implemented using the new function. And inets does not have to use the httpd_util  module internally.

**Comment by josevalim** [comment]
Should we name the functions `calendar:weekday_abbreviation/1` and `calendar:month_abbreviation/1`? Or `calendar:weekday_abbr/1` and `calendar:month_abbr()`? I believe it will make it clearer that it is not returning the full names for those reading the code.

File: lib/dialyzer/test/r9c_SUITE_data/src/inets/httpd_response.erl [hunk 1]
```diff
@@ -317,7 +317,7 @@ transform({content_type,undefined})->
     ["Content-Type:text/plain\r\n"];
 
 transform({date,undefined})->
```
Comment (IngelaAndin) [inline lib/dialyzer/test/r9c_SUITE_data/src/inets/httpd_response.erl:319]: @uabboli  Please comment on how you want to handle this test code.

Comment (uabboli) [reply lib/dialyzer/test/r9c_SUITE_data/src/inets/httpd_response.erl:319]: Modify the code any way you like to get rid of the warning(s).

**Comment by ao-song** [comment]
Close this PR as we think need more investigation in a better date format both for http and calendar, the code will not be merged for now, thanks for your valuable comments!

**Pull request closed without merging**